content/blog/2021-01-22-holiday-surveys.html

```diff
@@ -114,7 +115,7 @@ <h2>Travel and Other Social Behaviors During US Holidays</h2>
   filter(geo_value %in% str_to_lower(statelist)) %&gt;%
   mutate(value = plyr::mapvalues(geo_value, str_to_lower(statelist), regions),
          value = as.integer(factor(value)))
-
+attr(regionmap, &quot;metadata&quot;) &lt;- list(geo_type = &quot;state&quot;)
```

Comment:
this is weird: it was added in https://github.com/cmu-delphi/www-main/pull/706/files#diff-1fa4417df3994a017fb9ca08ed2a579d658b7167ccdae44b295a9e3aa3bf0880R111 but hasnt shown up til now...  i guess we havent actually re-rendered this page in over a year?  [no action needed; just wanted to point this out for posterity]